src/App/ElementMap.cpp

```diff
@@ -562,6 +562,11 @@ MappedName ElementMap::setElementName(const IndexedName& element, const MappedNa
         }
     }
 
+    // Originally in ComplexGeoData::setElementName
+    // LinkStable/src/App/ComplexGeoData.cpp#L1631
+    // if(!_ElementMap)
+    //     resetElementMap(std::make_shared<ElementMap>());
+
```

Comment:
Exactly.

`_ElementMap` (now `_elementMap`) is an `ElementMapPtr` object.
In RT fork it was accessible because the class it belongs to was in `ComplexGeoData`, now it isn't anymore (the closest object to perform this check could be `this` but it doesn't make much sense and, above all, it cannot be modified while the method is running).

About the comment, sure, a couple of words added to better explain are always a good thing IMO.